k8s/ldata/cluster-00-overlay/kustomization.yaml

```diff
@@ -1,2 +1,33 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+bases:
+  - ../../namespaces/admin/flux
```

Comment:
you need secrets for quite a few of these to work, should all be in the readme

Reply:
done